src/Message/RefundResponse.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Omnipay\CheckoutCom\Message;
+
+use Omnipay\Common\Message\ResponseInterface;
+
+class RefundResponse implements ResponseInterface
+{
+
+	private $data;
+	private $request;
+
+	public function __construct(RefundRequest $request, array $data)
+	{
+		$this->data = $data;
+		$this->request = $request;
+	}
+
+	public function getData()
+	{
+		return $this->data;
+	}
+
+	public function getRequest()
+	{
+		return $this->request;
+	}
+
+	public function isSuccessful()
+	{
+		if (!isset($this->getData()["action_id"])) {
+			return false;
+		}
+	}
```

Comment:
this function never returns true. Do you mean?
```suggestion
	public function isSuccessful()
	{
		return isset($this->getData()["action_id"]));
	}
```